core/app/modules/kit-library/assets/js/pages/overview/overview-taxonomy-badge.js

```diff
@@ -0,0 +1,35 @@
+import useTaxonomies from '../../hooks/use-taxonomies';
+import { useMemo } from 'react';
+import { Badge } from '@elementor/app-ui';
+import { Link } from '@reach/router';
+
+export default function OverviewTaxonomyBadge( props ) {
+	const { data } = useTaxonomies();
+
+	const type = useMemo(
+		() => {
+			if ( ! data ) {
+				return null;
+			}
+
+			return data.find( ( item ) => item.text === props.children )?.type;
+		},
+		[ data, props.children ]
+	);
+
+	if ( ! type ) {
+		return '';
+	}
+
+	return (
+		<Link to={ `/kit-library?taxonomies[${ type }][]=${ props.children }` }>
+			<Badge>
+				{ props.children }
+			</Badge>
+		</Link>
+	);
```

Comment:
A suggestion:

Kinda weird to me, passing the `props.children` as a query param.
I mean, I know why you did this, but maybe add that as a variable, just for readability? 

```suggestion
      const taxonomy = props.children;
  
	return (
		<Link to={ `/kit-library?taxonomies[${ type }][]=${ taxonomy }` }>
			<Badge>
				{ props.children }
			</Badge>
		</Link>
	);
```

And also in the `useMemo` find.